test/llvm/PREDIFF

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+TESTNAME=$1
+OUTFILE=$2
+
+TMPFILE="$outfile.prediff.tmp"
+
+FILECHECK=${CHPL_HOME}/third-party/llvm/install/linux64-gnu/bin/FileCheck
+
+mv $OUTFILE $TMPFILE
+$FILECHECK --input-file $TMPFILE $TESTNAME.chpl 2> $OUTFILE
+rm $TMPFILE
```

Comment:
I'd be comfortable merging this PR as-is or with the symbolic link. I think we should adjust the test system in a separate PR.